src/Servers/IIS/AspNetCoreModuleV2/IISLib/base64.cpp

```diff
@@ -176,11 +186,20 @@ constexpr auto NA = (255);
     BYTE    b0, b1, b2, b3;
     BYTE *  pbDecodeBuffer = (BYTE *) pDecodeBuffer;
 
-    cchEncodedSize = (DWORD)wcslen(pszEncodedString);
-    if (NULL != pcbDecoded) {
+    if (NULL != pcbDecoded)
```

Comment:
I imagine this is a silly question, but which `NULL`s became `nullptr`s?

Reply:
All of them. But I stopped once I realized how many there actually were.